src/Controllers/ContentController.php

```diff
@@ -223,6 +223,7 @@ public function getRecommended(Request $request)
     public function getContentOptimised($contentId, Request $request, $playlistItemId = null)
     {
         array_push(ContentRepository::$availableContentStatues, ContentService::STATUS_ARCHIVED);
+        array_push(ContentRepository::$availableContentStatues, ContentService::STATUS_UNLISTED);
```

Comment:
Should we not include both archived and unlisted?
```suggestion
        array_push(ContentRepository::$availableContentStatues, ContentService::STATUS_UNLISTED, ContentService::STATUS_ARCHIVED));
```